src/components/UploadEditor.vue

```diff
@@ -103,6 +105,16 @@ export default {
 		addMoreAriaLabel() {
 			return t('spreed', 'Add more files')
 		},
+		// FIXME: use real flag for voice message
+		// Hide the plus button in case this editor is used while sending a voice
+		// message
+		isVoiceMessage() {
+			const firstFile = this.files[Object.keys(this.files)[0]]
+			if (!firstFile) {
+				return false
+			}
+			return firstFile.temporaryMessage.messageParameters.file.name.endsWith('mp3')
```

Comment:
You need to save differently that a voice recording was done.
Now when I drop a single mp3 first in the chat, I can not add more files 🙈

Reply:
I'm on it https://github.com/nextcloud/spreed/pull/5610/files/855f0818d78b1a3c8b6d930114c67d710bc5928d#diff-ece8b8cb7a642dac2cc86be297c490fb4cc34df9dc3d353d6f306e4910227ec2R108-R110